.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,65 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+## Describe the bug
+A clear and concise description of what the bug is.
+
+## To Reproduce
+Please provide a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example).
+
+>    …Minimal – Use as little code as possible that still produces the same problem
+>   …Complete – Provide all parts someone else needs to reproduce your problem in the question itself
+>   …Reproducible – Test the code you're about to provide to make sure it reproduces the problem
+ 
+
+## Expected behavior
+A clear and concise description of what you expected to happen.
+
+## Screenshots
+If applicable, add screenshots to help explain your problem.
+
+# Environment (please complete the following information):
+
+## Java/JDK
+`java -version`
+
+## Groovy version
+>__Note that versions older than 2.5 are no longer supported.__
+
+`groovy -version`
+
+## Build tool version
+
+### Gradle
+`gradle -version`
```

Comment:
Similar to Groovy, this will usually give the system installed Gradle version.
As it is highly recommended that any project uses the Gradle wrapper and almost all I hit in the wild do, I'd suggest we show `gradlew` here.